numpy/linalg/lapack_lite/f2c_lapack.f.patch

```diff
@@ -0,0 +1,48 @@
+@@ -267,8 +267,8 @@
+      Scan up each column tracking the last zero row seen.
+          ILACLR = 0
+          DO J = 1, N
+             DO I = M, 1, -1
+-               IF( A(I, J).NE.ZERO ) EXIT
+-            END DO
++               IF( A(I, J).NE.ZERO ) GOTO 10
++   10       END DO
```

Comment:
I don't think this is correct for the EXIT, what you have here is equivalent to the C continue, whereas you want a break, i.e., GOTO the first statement after the END DO. See https://gcc.gnu.org/onlinedocs/gcc-3.4.4/g77/CYCLE-and-EXIT.html .